frontend/src/metabase/visualizations/components/settings/ChartSettingOrderedColumns.jsx

```diff
@@ -59,9 +59,11 @@ export default class ChartSettingOrderedColumns extends Component {
   };
 
   handleSortEnd = ({ oldIndex, newIndex }) => {
-    const fields = [...this.props.value];
-    fields.splice(newIndex, 0, fields.splice(oldIndex, 1)[0]);
-    this.props.onChange(fields);
+    const enabledColumns = [...this.props.value].filter(columnSetting =>
+      findColumnForColumnSetting(this.props.columns, columnSetting),
+    );
```

Comment:
I think I can safely use
```suggestion
    const enabledColumns = [...this.props.value].filter(columnSetting =>
      columnSetting.enabled
    );
```
silimiarly to what I've done in `frontend/src/metabase/visualizations/visualizations/Table.jsx`, but I use this since we have similar logic that yield a variable named `enabledColumns` in the `render` function.